src/camera/camera2d.jl

```diff
@@ -1,9 +1,9 @@
 struct Camera2D <: AbstractCamera
-    area::Observable{Rect2f}
-    zoomspeed::Observable{Float32}
+    area::Observable{Rect2{Float64}}
```

Comment:
```suggestion
    area::Observable{Rect2e}
```
or do we want to move away from f's and e's entirely in the internals?